content/docs/command-reference/status.md

```diff
@@ -13,82 +13,68 @@ usage: dvc status [-h] [-v] [-j <number>] [-q] [-c] [-r <name>] [-a] [-T]
-| ------ | -------------- | --------------------------------------------------------------------------------------------------------------------------- |
-| local  | _none_         | Comparisons are made between data files in the workspace and corresponding files in the cache directory (e.g. `.dvc/cache`) |
-| remote | `--remote`     | Comparisons are made between the cache, and the given remote. Remote storage is defined using the `dvc remote` command.     |
-| remote | `--cloud`      | Comparisons are made between the cache, and the default remote, typically defined with `dvc remote --default`.              |
-
-DVC determines which data and code files to compare by analyzing all stages (in
-`dvc.yaml` and `.dvc` files in the <abbr>workspace</abbr> (the `--all-branches`
-and `--all-tags` options compare multiple workspace versions).
-
-The comparison can be limited to certain stages (in `dvc.yaml`) or `.dvc` files
-only, by listing them as `targets`. (Changes are reported only against these.)
-When this is combined with the `--with-deps` option, a search is made for
-changes in other stages that affect each target.
-
-In the local mode, changes are detected through the hash value of every file
-listed in every stage (in `dvc.yaml` or `.dvc` files) in question against the
-corresponding file in the file system. The command output indicates the detected
-changes, if any. If no differences are detected, `dvc status` prints this
-message:
-
-```dvc
-$ dvc status
-Data and pipelines are up to date.
-```
-
-This indicates that no differences were detected, and therefore no stages would
-be executed by `dvc repro`.
-
-If instead, differences are detected, `dvc status` lists those changes. For each
-stage with differences, the changes in <abbr>dependencies</abbr> and/or
+(implying `dvc push` or `dvc pull` should be run to synchronize them). The
+_remote_ mode is triggered by using the `--cloud` or `--remote` options:
+
+| Mode   | Option     | Description                                                                                                                 |
+| ------ | ---------- | --------------------------------------------------------------------------------------------------------------------------- |
+| local  | _none_     | Comparisons are made between data files in the workspace and corresponding files in the cache directory (e.g. `.dvc/cache`) |
+| remote | `--remote` | Comparisons are made between the cache, and the given remote. Remote storage is defined using the `dvc remote` command.     |
+| remote | `--cloud`  | Comparisons are made between the cache, and the default remote (typically defined with `dvc remote --default`).             |
+
+Without arguments, this command checks all stages (in `dvc.yaml`) and `.dvc`
+files to compare the hash values of their <abbr>outputs</abbr> (in `dvc.lock`)
```

Comment:
(for stages output hashes as defined in `dvc.lock`)?

Reply:
"checks stages in `dvc.yaml` for stage output hash values as defined in `dvc.lock`" ? That's contradictory. Let me think 🧠

Reply:
Changed to 

"checks all stages (defined in `dvc.yaml`) and `.dvc` files, and compares the hash values of their <abbr>outputs</abbr> (found in `dvc.lock` for stages) against"

in acf0196f.

Reply:
#1663 can probably simplify this too.